docs/maui/markup/extensions/flex-layout-extensions.md

```diff
@@ -0,0 +1,111 @@
+---
+title: FlexLayout extensions - .NET MAUI Community Toolkit
+author: danielftz
+description: The FlexLayout extensions provide a series of extension methods that support positioning Views in FlexLayouts.
+ms.date: 04/06/2022
+---
+
+# FlexLayout extensions
+
+[!INCLUDE [docs under construction](../../includes/preview-note.md)]
+
+The FlexLayout extensions provide a series of extension methods that support positioning `View`s in `FlexLayout`s.
+
+The extensions offer the following methods:
+
+## AlignSelf
+
+The `AlignSelf` extension method allows you to set how a `View` in `FlexLayout` is aligned on the cross axis. Setting this property overrides the `AlignItems` property set on the parent `FlexLayout` itself. For further detail refer to the [Microsoft documentation](/dotnet/maui/user-interface/layouts/layouts/flexlayout#alignself).
+
+## Basis
+
+The `Basis` extension method allows you to set the amount of space that's allocated to a `View` in `FlexLayout` on the main axis. The size can be specified in device-independent units, as a percentage of the size of the `FlexLayout` or based on the `View`'s requested width or height. For further detail refer to the [Microsoft documentation](/dotnet/maui/user-interface/layouts/flexlayout#basis).
+
+## Grow
+
+The `Grow` extension method specifies the amount of available spacea `View` in `FlexLayout` should use on the main axis. For further detail refer to the [Microsoft documentation](/dotnet/maui/user-interface/layouts/flexlayout#grow).
+
+## Order
+
+The `Order` extension method allows you to change the order that the children of the FlexLayout are arranged.  Setting this property overrides the order that it appears in the `Children` collection. For further detail refer to the[Microsoft documentation](/dotnet/maui/user-interface/layouts/flexlayout#order).
+
+## Shrink
+
+The `Shrink` extension method allows you to indicate which `View` in `FlexLayout` is given priority in being displayed at their full sizes when the aggregate size of `Children` is greater than 
+on the main axis. For further detail refer to the[Microsoft documentation](/dotnet/maui/user-interface/layouts/flexlayout#shrink).
+
+
+## Syntax
+
+```csharp
+using Microsoft.Maui.Layouts;
+using CommunityToolkit.Maui.Markup;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Graphics;
+
+namespace CommunityToolkit.Maui.Markup.Sample.Pages;
+
+public class FlexLayoutSamplePage : ContentPage
```

Comment:
Would it make sense to divide up the sample code a bit more and put it under the specific headings like we do for [other Markup docs](https://docs.microsoft.com/dotnet/communitytoolkit/maui/markup/extensions/bindable-object-extensions)?

@brminnick opinions?

Reply:
> In addition to the other comments, make sure to add an entry in the TOC file for this!

I'm not sure where in TOC file I would add it

Reply:
See this PR as an example: https://github.com/MicrosoftDocs/CommunityToolkit/pull/50/files